authentication_handlers_test.go

```diff
@@ -326,7 +326,7 @@ func TestAuthenticationUpdateNotFound(t *testing.T) {
 	c.SetParamValues("not existing uid")
 	c.Request().Header.Add("Content-Type", "application/json;charset=utf-8")
 
-	notFoundAuthenticationUpdate := ErrorHandlingContext(AuthenticationUpdate)
+	notFoundAuthenticationUpdate := ErrorHandlingContext(AuthenticationEdit)
```

Comment:
maybe we can change the names of variable on left side here to `notFoundAuthenticationEdit`
and same on other places in tests

Reply:
that's right - fixed, thanks!